api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryutil', 'responsetime', 'throughput'];
```

Comment:
@pradyutsarma  it is already in production?  I can keep both if your customer are seriously use it.